docker-compose-e2e-readme.md

```diff
@@ -0,0 +1,98 @@
+# Opzetten/Opstarten containerized BRP Bevragen end-to-end test omgeving
+
+Start de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml up -d
+```
+
+Stop de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml down
+```
+
+## Beheren van de BRP bevragen test database
+
+Dit kan worden gedaan met behulp van de pgAdmin tool. Deze wordt opgestart door in een browser naar de volgende url te navigeren
+
+```
+http://localhost:5433
+```
+
+login met Email Address / Username: admin@haalcentraal.nl en Password: root
+
+### Connectie maken met de database
+
+rechter muis klik op Servers. Selecteer Register > Server...
+
+![Register Server](./docs/img/pgAdmin-register-server.png)
+
+
+In de Register-Server dialog box, vul de volgende velden in
+
+![Register Server dialog box](./docs/img/pgAdmin-register-server-dialog.png)
+
+password: root
+
+klik op de **Save** button
+
+De volgende stappen zijn alleen nodig als de BRP database nog niet is ge-restored
+
+### Aanmaken Login/Group Role 'haalcentraal'
+
+rechter muis klik op Login/Group Roles. Selecteer Create > Login/Group Role...
+
+![Create Login/Group Role](./docs/img/pgAdmin-create-login-role.png)
+
+In de Create - Login/Group Role dialog box, vul de Name veld en selecteer alle privileges
+
+![Create Login/Group Role dialog box](./docs/img/pgAdmin-create-login-role-dialog.png)
+
+klik op de **Save** button
+
+### Aanmaken 'rvig_haalcentraal_testdata' database
+
+rechter muis klik op Databases. Selecteer Create > Database...
+
+![Create Database](./docs/img/pgAdmin-create-database.png)
+
+In de Create - Database dialog box, vul de volgende velden in
+
+![Create Database dialog box](./docs/img/pgAdmin-create-database-dialog.png)
+
+klik op de **Save** button
+
+### Restore 'rvig_haalcentraal_testdata' database
+
+rechter muis klik op rvig_haalcentraal_testdata. Selecteer Restore...
+
+![Restore Database](./docs/img/pgAdmin-restore-database.png)
+
+In de Restore (Database: rvig_haalcentraal_testdata) dialog box, klik op de map icoon in de Filename text box
+
+![Restore Database dialog box](./docs/img/pgAdmin-restore-database-dialog.png)
+
+In de Select file dialog box, klik op de ... knop en selecteer Upload
+
+![Restore Database](./docs/img/pgAdmin-restore-database-upload-file.png)
+
+Drag en Drop de rvighaalcentraal.sql bestand in de Select file dialog box
```

Comment:
Dit bestand stel je nu niet ter beschikking in deze pull request? Of mis ik iets?
Wat kan iemand hiermee wanneer die de database niet heeft?

kan je geen docker maken wat de database al in zit?

Reply:
wanneer we een update krijgen, bijv. omdat er een nieuwe versie is van de proxy, moet ik dan al deze stappen opnieuw doen?